lib/adal-angular.js

```diff
@@ -225,7 +226,7 @@ if (typeof module !== 'undefined' && module.exports) {
                             _adal._saveItem(_adal.CONSTANTS.STORAGE.START_PAGE_PARAMS, JSON.stringify(toParams));
 
                             _adal.info('State change event for:' + $location.$$url);
-                            loginHandler();
+                            loginHandler('adal:unauthenticated&renew_inactive');
```

Comment:
this is the same string as another, please diffrentiate it a bit so we can tell which is which